assets/src/components/Sidebar/CampaignSidebar.js

```diff
@@ -111,6 +111,15 @@ export class CampaignSidebar extends Component {
       ) {
         const theme = await loadTheme(themeName);
         this.setState({ theme: theme });
+        if (themeName) {
+          try {
+            const response = await fetch(`${ themeJsonUrl + themeName.replace('-new', '') }.json`);
```

Comment:
We'll probably migrate content to a name without `-new` now that these are available again. After that we can clean up this logic, and some similar other ones throughout the code.